apis/bigtable/v1alpha1/types.generated.go

```diff
@@ -12,244 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package v1alpha1
-
-import (
-	refs "github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
-)
-
-// +kcc:proto=google.bigtable.admin.v2.AutoscalingLimits
-type AutoscalingLimits struct {
```

Comment:
Oops, the removal of existing types was an unintended side effect. I'm modernizing the generated types for Bigtable v1alpha1 in #4700. If you pick up #4700 and run the following command, the changes should be additive only.

```
go run . generate-types \
  --service google.bigtable.admin.v2 \
  --api-version bigtable.cnrm.cloud.google.com/v1alpha1  \
  --resource BigtableAuthorizedView:AuthorizedView \
  --resource BigtableBackup:Backup \
  --resource BigtableCluster:Cluster \
  --resource BigtableLogicalView:LogicalView
```